website/docs/r/spring_cloud_gateway_route_config.html.markdown

```diff
@@ -74,6 +75,10 @@ The following arguments are supported:
 
 * `open_api` - (Optional) One or more `open_api` blocks as defined below.
 
+* `protocol` - (Optional) Specifies the protocol of routed Spring Cloud App. Allowed values are `HTTP` and `HTTPS`. Defaults to `HTTP`. 
+
+~> **Note:** `protocol`'s default value `HTTP` is insecure.
```

Comment:
Can we rephrase this to 
```suggestion
~> **Note:** You likely want to use `HTTPS` in a production environment, since `HTTP` offers no encryption.
```